doc/lua_api.md

```diff
@@ -10685,6 +10685,11 @@ Used by `minetest.add_particle`.
 
     drag = {x=0, y=0, z=0},
     -- v5.6.0 and later: Optional drag value, consult the following section
+    -- Note: Only a vector is supported here. Alternative forms like a single
+    -- number are not supported.
+
+    jitter = {min = ..., max = ..., bias = 0},
```

Comment:
What about `attract`, `die_on_contact`, etc. from "List of particlespawner properties"?

It seems that there's no clear differentiation between single particles and particlespawners. If they do not share all properties, it might make more sense to list *which* properties they accept, and use a different title for "List of particlespawner properties" (e.g. "Particle/ParticleSpawner fields definition")

Reply:
> What about `attract`, `die_on_contact`, etc. from "List of particlespawner properties"?

These are not supported by single particles and are therefore not mentioned here.
  
(Also note that `die_on_contact` is actually `attract.die_on_contact` (also documented))

> It seems that there's no clear differentiation between single particles and particlespawners.

Currently, the properties supported for single particles are listed in this section ("Particle definition") and the properties supported for particlespawners are listed in the next one ("ParticleSpawner definition"). While this could definitely use some reorganisation and unification, I won't do that in this PR. In this PR, I only fixed incorrect and/or missing documentation.

Reply:
> (Also note that die_on_contact is actually attract.die_on_contact (also documented))

You're right. I did not notice the difference of indent levels plain text view, whereas they'd have different list point styles when rendered as pretty HTML.